src/pages/kb/user-guide/querying/query-parameters.md

```diff
@@ -120,3 +118,33 @@ WHERE org_id = {{org_id}} AND created_at > '{{start_date}}'
 ```
 
 We use two parameters: `{{org_id}}` and `{{start_date}}`.
+
+## Parameter Mapping on Dashboards
+
+Query Parameters can also be powerfully controlled within dashboards. You can link together parameters on different widgets, set static parameter values, or choose values individually for each widget.
+
+You select your desired parameter mapping when adding dashboard widgets that depend on a parameter value. Each parameter in the underlying query will appear in the **Parameters** list.
+
+<img src="/assets/images/docs/gitbook/dashboard_parameter_mapping.png" width="100%">
+
+{% callout info %}
+You can also access the parameter mapping interface by clicking the three dots on the top right of a dashboard widget, then clicking `Edit Parameters`.
+{% endcallout %}
+
++ **Title** is the display name for your parameter and will appear beside the value selector on your dashboard. It defaults to the parameter keyword (see next bullet). Edit it by clicking the pencil glyph. Note that a titles are not displayed for static dashboard parameters because the value selector is hidden. If you select `Static value` as your Value Source then the Title field will be grayed out.
+
++ **Keyword** is the string literal for this parameter in the underlying query. This is useful for debugging if your dashboard does not return expected results.
+
++ **Default Value** is what Redash will pass to the database the next time you refresh this dashboard.
```

Comment:
If I'm not mistaken, it's called default value cause the eventual value may be trampled by querystring param, which means it's not necessarily the what would be passed.

Reply:
That is really good to know. Will update and commit.

Reply:
@kravets-levko am I right? (need to make sure)

Reply:
Exactly. It's the value Redash will use if you don't provide any other value.